runtime/array.c

```diff
@@ -513,6 +513,58 @@ CAMLprim value caml_array_concat(value al)
   return res;
 }
 
+CAMLprim value caml_array_concat_sub(value al)
+{
+  value static_arrays[STATIC_SIZE], * arrays;
```

Comment:
STATIC_SIZE is defined "inside" caml_array_concat, should it be moved above now that it's used in multiple functions.

Reply:
Done